### PR TITLE
feat(vue3): upgrade to vue 3

### PR DESCRIPTION
**Pull request by floryst**
- Upgrades vuetify 3
- Upgrades vue 3
- Drops usePanel, since vuetify 3's panels are key-based
- MeasurementsList is removed in favor of tool-specific components (e.g. RulerList, RectangleList).
- nav drawer is no longer resizable
- Theming options are available. Note that the light theme is not well-tested.
- The color function range slider has been replaced with a shift + width slider duo. I haven't been able to port over the old ColorFunctionSlider from vuetify 2.
- Refactoring some reactivity usage.
- Some minor clean-up as I go through the codebase

### CODE REVIEW

**Comment by floryst** [comment]
I think this will be ready to test and evaluate after the rectangle tool gets merged.

**Comment by floryst** [comment]
This is ready to evaluate. There are a lot of changes in this PR, partly due to vuetify/vue 3 upgrades and partly due to some refactoring in order to address some subtle bugs that appeared when upgrading.

A code review would be nice, but given the size of this PR a skim should be sufficient. What is important is that the application preview operates as expected.

**Review by PaulHax — COMMENTED** [review]
Making my dreams come true with this PR!

One issue is the 2D slice window-level? is not always applied when I:
1. Load Prostate and Fetus samples
2. Click on Fetus in DataBrowser
3. Click on Protstate in DataBrowser
4. Repeat 2 and 3.
5. Fetus alternates between light and dark on each load.

Another issue:
Volume center of rotation is not in center with Fetus.

I'll keep poking around...

File: src/components/PatientStudyVolumeBrowser.vue [hunk 1]
```diff
@@ -260,45 +251,36 @@ export default defineComponent({
...
-                                  color="grey lighten-5"
-                                />
-                                <v-icon v-else>{{ volume.layerIcon }}</v-icon>
-                              </v-btn>
-                            </template>
-                            {{ volume.layerTooltip }}
-                          </v-tooltip>
+                          <div class="layer-btn-container">
+                            <v-btn
+                              v-show="volume.layerable"
```
[the remark this answers is not in the document]
Comment (floryst) [reply src/components/PatientStudyVolumeBrowser.vue:263]: Sounds good!

**Comment by floryst** [comment]
Summary of latest commits:
- splits out view config store into several different stores.
- fixes the aforementioned window-level bug
- fixes the aforementioned rotation bug

**Review by PaulHax — COMMENTED** [review]
The DICOMWeb data browser components have runtime errors.  I'll open a PR on this one to fix.

**Comment by PaulHax** [comment]
Another window level issue:
1. Load Volume
2. Load another volume
3. Adjust window level <- W/L range goes to 0-1 and image gets "blown out"

File: src/components/PatientBrowser.vue
[the remark this answers is not in the document]
Comment (floryst) [reply src/components/PatientBrowser.vue:1]: Good catch. This looks to be a timing thing where we want the component to unmount, but the computed properties are evaluated first.

**Comment by floryst** [comment]
Summary of recent commits:
- sync window/level min/max values
- guard against deleted patient
- some style adjustments

@sankhesh @aylward FYI this is ready to evaluate. The deploy preview link is provided above.

**Comment by sankhesh** [comment]
Looks good overall. The warning box for volume quality slider is orange and too in the face. In the current version, it has a color set on it - https://github.com/Kitware/VolView/blob/dd5f9e305b68057600a3978af350b14329235800/src/components/VolumeProperties.vue#L104-L112

**Comment by floryst** [comment]
Good point. I'll tone down the colors a bit.

**Comment by floryst** [comment]
I've adjusted some styles and improved the About popup.

**Comment by floryst** [comment]
FYI all of these commits will be squashed prior to merging.

**Comment by floryst** [comment]
This seems good to go. We can revert if needed.

**Pull request merged**